HW3/Menues/Database/Books.cs

```diff
@@ -21,18 +21,18 @@ public void Create(SqlConnection connection)
 
             ConsoleHelper.WriteService("Enter library of the book");
             string input = Console.ReadLine();
-            if(!int.TryParse(input, out int id) && id > 0)
-            {
+            if(!int.TryParse(input, out int id) && id > 0) // it seems like broken logic. If we can parse number, we won't check the id > 0. If we cannot parse number, then we check if int default > 0. 
```

Comment:
to lines 20, 23 and so on:
better separate logic between methods; so 'Create' should create record in DB, it shouldn't check users' inputs; I think, receiving data for query is better move to other method

Reply:
Done